chain/client/src/chunk_producer.rs

```diff
@@ -239,10 +240,14 @@ impl ChunkProducer {
         debug!(target: "client", me = ?validator_signer.validator_id(), next_height, ?shard_id, "Producing chunk");
 
         let shard_uid = shard_id_to_uid(self.epoch_manager.as_ref(), shard_id, epoch_id)?;
-        let chunk_extra = self
-            .chain
-            .get_chunk_extra(&prev_block_hash, &shard_uid)
-            .map_err(|err| Error::ChunkProducer(format!("No chunk extra available: {}", err)))?;
+        let chunk_extra = if cfg!(feature = "protocol_feature_spice") {
+            // TODO(spice): using default values as a placeholder is a temporary hack
```

Comment:
optional nit: it may be good to include in todos what needs to be done in addition to why. For example, refactor to avoid using default values for chunk extra.